Formula/r/ratchet.rb

```diff
@@ -1,8 +1,9 @@
 class Ratchet < Formula
   desc "Tool for securing CI/CD workflows with version pinning"
   homepage "https://github.com/sethvargo/ratchet"
-  url "https://github.com/sethvargo/ratchet/archive/refs/tags/v0.9.0.tar.gz"
-  sha256 "e6d418075866a0c1040ef36a009cb4f8d2ed0edf6f4b5cb0041c3d076c6a349a"
+  url "https://github.com/sethvargo/ratchet.git",
+    tag:      "v0.9.0",
+    revision: "8f1cce74d9a2fe51ff550a51b1a175d913981b7a"
```

Comment:
Let's align the arguments to `url`.
```suggestion
  url "https://github.com/sethvargo/ratchet.git",
      tag:      "v0.9.0",
      revision: "8f1cce74d9a2fe51ff550a51b1a175d913981b7a"
```

Reply:
Done